### DISCUSSION TITLE
sorry, something went wrong

### DISCUSSION THREAD

**Discussion by GusGF** [community/community#44600] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I click on my profile pic to get to settings I'm greeted with this message. I've tried Firefox and Chrome and I'm on the latest version of Ubuntu. Does anyone know what the issue might be?

**Comment by Ian-sc-Kang** [comment]
same issue here
![image](https://user-images.githubusercontent.com/87746894/212974254-94099923-90eb-49cf-9fe2-e566cc258ee8.png)

**Reply by GusGF** [reply]
That's exactly what I'm getting. I'm a new user of Linux so thought I was doing something wrong, maybe I am. But I've logged in and authenticated so??

**Reply by alec-jensen** [reply]
it seems to be an outage on githubs end

**Reply by GusGF** [reply]
Okay, well spotted

**Reply by GusGF** [reply]
https://www.githubstatus.com/ says all is normal?!

**Reply by Ian-sc-Kang** [reply]
alright! everything seems back to normal

**Reply by GusGF** [reply]
Yes I can confirm working for me too

**Comment by alec-jensen** [comment]
same issue here

**Reply by GusGF** [reply]
I'm dual booting with Win10 but I need to get this working on Ubuntu so the fact it works on W10 won't help me.

**Comment by alec-jensen** [comment, marked as answer]
if you are trying to access settings, go to this link: https://github.com/settings/profile
github has some outage that stops the menu from opening

**Reply by GusGF** [reply]
Excellent my good man!!
:)
